docs/README.md

```diff
@@ -36,8 +36,7 @@ You need to have [Ruby](https://www.ruby-lang.org/en/documentation/installation/
 installed. Also install the following libraries:
```

Comment:
Do you want to mention Python 3 is required here? or does it still work with Python 2?

Reply:
I think it still works with Python 2 because it seems not using Python at all. But I guess we should better stick to Python 3 ..